autoload/clever_f.vim

```diff
@@ -96,6 +101,43 @@ function! s:is_timedout() abort
     return elapsed_ms > g:clever_f_timeout_ms
 endfunction
 
+function! s:mark_direct(forward, count) abort
+    let line = getline('.')
+    let c_start = col('.')
+    let c_end = col('.')-2
+    let l = line('.')
+
+    if (a:forward && c_start+1 > len(line)-1) ||
+        \(!a:forward && c_end+1 < 0)
```

Comment:
What does this check?

Reply:
checking if there is no chars forward (backward)